src/components/my-imports/import-list.tsx

```diff
@@ -188,7 +189,8 @@ export class ImportList extends React.Component<IProps, IState> {
     return (
       <div>
         <div>
-          {item.name} {item.version ? 'v' + item.version : ''}
+          <span data-cy='item-name'> {item.name} </span>{' '}
```

Comment:
```suggestion
          <span data-cy='item-name'>{item.name}</span>{' '}
```

otherwise the space would offset the padding